File: Beginner/Coder Life Matters (CODERLIF)/coder.py
t = int(input())


while t:
    A = list(map(int, input().split()))
    c = 0

    for i in A:
        if i==1:
            c = c+1
        else:
            c = 0
        if c>5:
            print('#coderlifematters')
            break
    
    if c<6:
        print('#allcodersarefun')

    t = t-1

File: Beginner/Coder Life Matters (CODERLIF)/coder.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin>>t;
	while(t--) {
	    int S[30];
	    for(int i=0;i<30;i++) {
	        cin>>S[i];
	    }
	    int c =0 ;
	    for(int i=0;i<30;i++) {
	        if(S[i] == 1) {
	            c++;
	            if(c > 5) {
	                cout<<"#coderlifematters"<<endl;
	                break;
	            }
	        }
	        else{
	            c=0;
	        }
	    }
	    if(c <= 5) {
	        cout<<"#allcodersarefun"<<endl;
	    }
	}
	return 0;
}
